packages/server/models/Theme.ts

```diff
@@ -0,0 +1,30 @@
+import { Column, DataType, Model, Table } from 'sequelize-typescript'
+
+@Table({
+  timestamps: false,
+  paranoid: true,
+  tableName: 'theme',
+})
+export class Theme extends Model<Theme> {
+  @Column({
+    type: DataType.INTEGER,
+    field: 'id_theme',
+    primaryKey: true,
+    autoIncrement: true,
+  })
+  override id!: number
+
+  @Column({
+    type: DataType.INTEGER,
+    field: 'user_id',
+    allowNull: false,
+  })
+  userId!: number
+
+  @Column({
+    type: DataType.STRING,
+    field: 'name_theme',
```

Comment:
[Предложение]
И если менять порядок для theme_id, то тут
```suggestion
    field: 'theme_name',
```

Reply:
fix